x-pack/plugins/cases/public/components/case_action_bar/index.tsx

```diff
@@ -166,7 +166,7 @@ const CaseActionBarComponent: React.FC<CaseActionBarProps> = ({
                       data-test-subj="case-refresh"
                       flush="left"
                       iconType="refresh"
-                      onClick={onRefresh}
+                      onClick={refreshCaseViewPage}
```

Comment:
instead we'll be calling this `refreshCaseViewPage` in any case where the previous props were used.